.github/workflows/release.yml

```diff
@@ -61,12 +65,14 @@ jobs:
       matrix:
         python-version: ['3.5', '3.6', '3.7']
     steps:
+      - if: github.event_name == 'schedule'
+        run: echo "::set-env name=NIGHTLY_FLAG::--nightly"
```

Comment:
As just a sanity check that [this method](https://help.github.com/en/actions/reference/development-tools-for-github-actions#set-an-environment-variable-set-env) is working correctly could you do a quick commit to echo `$NIGHTLY_FLAG` for all three OS's? I would just modify the nightly flag to be added for py35 and then we can revert that commit.

Reply:
Sure thing